pkg/controller/servicebinding/reconciler.go

```diff
@@ -31,6 +36,10 @@ const (
 	EmptyApplicationReason = "EmptyApplication"
 	// ApplicationNotFoundReason is used when the application is not found.
 	ApplicationNotFoundReason = "ApplicationNotFound"
+	// FailedToGatherServiceInformationReason is used when the it is failed to fetch backend service information.
+	FailedToGatherServiceInformationReason = "FailedToGatherServiceInformation"
+	// FailedToSaveSecretReason is used when the it is failed to save intermediate secret.
```

Comment:
Do we need this ? Could not find any reference to this particular FailedToSaveReason in this PR.

Reply:
removed